src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -48,6 +48,8 @@ using matrix::Vector2d;
 using matrix::Vector3f;
 using matrix::wrap_pi;
 
+static constexpr float MAX_ALT_REF_RATE_FOR_LEVEL_FLIGHT{0.1f};
```

Comment:
Please move to the header and use the convention of the FW position controller (kMax...)

Reply:
Moved in https://github.com/PX4/PX4-Autopilot/pull/22401/commits/d182e0dd742a988587540a592448559c5dc0d437